arch/arm/soc/st_stm32/stm32f0/Kconfig.defconfig.series

```diff
@@ -24,7 +24,7 @@ endif # GPIO_STM32
 
 if I2C && (I2C_1 || I2C_2 || I2C_3)
```

Comment:
Why not `if I2C_STM32`?

Reply:
why? I did like it's done for SPI abve it.

Reply:
Sorry, I posted it in the wrong file.